### PR TITLE
Improve edit content menu item wording

### PR DESCRIPTION
**Pull request by F10Ragger**
change name

### CODE REVIEW

File: packages/admin/cms-admin/src/pages/pageTree/PageContextMenu.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const PageContextMenu = (props: PageContextMenuProps): React.ReactElement => {
                         <ListItemIcon>
                             <Edit />
                         </ListItemIcon>
-                        <ListItemText primary={intl.formatMessage({ id: "comet.generic.content", defaultMessage: "Content" })} />
+                        <ListItemText primary={intl.formatMessage({ id: "comet.generic.editContent", defaultMessage: "Edit Content" })} />
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/pages/pageTree/PageContextMenu.tsx:103]:
```suggestion
                        <ListItemText primary={intl.formatMessage({ id: "comet.generic.editContent", defaultMessage: "Edit content" })} />
```

**Comment by johnnyomair** [comment]
The commit message and title of this PR are rather generic. It would be better if you would describe why it has been changed, and not that it has been changed.

For example

Improve edit content menu item wording
Using only "Content" for the edit content menu item is confusing for our users. We therefore change it to "Edit content" for better understanding.

File: packages/admin/cms-admin/src/pages/pageTree/PageContextMenu.tsx [hunk 2]
```diff
@@ -100,7 +100,7 @@ const PageContextMenu = (props: PageContextMenuProps): React.ReactElement => {
                         <ListItemIcon>
                             <Edit />
                         </ListItemIcon>
-                        <ListItemText primary={<FormattedMessage {...messages.content} />} />
+                        <ListItemText primary={intl.formatMessage({ id: "comet.generic.editContent", defaultMessage: "Edit content" })} />
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/pages/pageTree/PageContextMenu.tsx:103]:
```suggestion
                        <ListItemText primary={intl.formatMessage({ id: "comet.pages.pages.page.editContent", defaultMessage: "Edit content" })} />
```

This is not a generic message 😉

**Review by johnnyomair — APPROVED** [review]

File: packages/admin/cms-admin/src/pages/pageTree/PageContextMenu.tsx [hunk 3]
```diff
@@ -100,7 +100,7 @@ const PageContextMenu = (props: PageContextMenuProps): React.ReactElement => {
                         <ListItemIcon>
                             <Edit />
                         </ListItemIcon>
-                        <ListItemText primary={<FormattedMessage {...messages.content} />} />
+                        <ListItemText primary={intl.formatMessage({ id: "comet.pages.pages.page.editContent", defaultMessage: "Edit content" })} />
```
Comment (dkarnutsch) [inline packages/admin/cms-admin/src/pages/pageTree/PageContextMenu.tsx:103]:
```suggestion
                        <ListItemText primary={<FormattedMessage id="comet.pages.pages.page.editContent" defaultMessage="Edit content" />} />
```

Please stick with the FormattedMessage

**Review by dkarnutsch — CHANGES_REQUESTED** [review]

**Review by dkarnutsch — APPROVED** [review]

**Pull request merged**